portfolio/src/main/java/com/google/sps/servlets/RandomPhraseServlets.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/random-phrase")
+public class RandomPhraseServlets extends HttpServlet {
+
+    String[] funFacts = {
```

Comment:
You can add "public static final" and reformat the name to be FUN_FACTS, as is typical for constants in Java.

Reply:
```suggestion
    public static final String[] funFacts = {
```